roles/buildbot-master/tasks/main.yml

```diff
@@ -163,14 +163,19 @@
     state: enabled
     permanent: true
     immediate: true
+    zone: "{{ item }}"
+  with_items:
+    - public
+    - dmz
   tags:
    - buildbot-master
 
-- name: permit port 9989 (workers) in firewall for buildbot
+- name: permit port 9989 (workers) in firewall for buildbot (dmz only)
   firewalld:
     port: 9989/tcp
     state: enabled
     permanent: true
     immediate: true
+    zone: dmz
```

Comment:
Does this *actually* change the current setup, or does it just add  a new rule that is dmz only?

Reply:
Additive only. We'd need the converse (zone: public, state: disabled) to do that. Which, we can do manually later once everyone is on the VPN. Or if you fire this at the test worker so it joins the VPN, we could make it connect to buildbot.zt.flathub.org already.